### ISSUE TITLE
Returning text/html

### ISSUE THREAD

**Issue by jamesharrington** [dfournier/plasticine#20]
It would be quite awesome to be able to return some plain text that we could use as html

**Comment by dfournier** [comment]
You're right, it would be a nice feature! I can do that in less than a week, would it suits you?

**Comment by jamesharrington** [comment]
That would be awesome!
